src/components/GoodList/GoodList.tsx

```diff
@@ -0,0 +1,13 @@
+import { GoodCard } from '../GoodCard/GoodCard';
+
+interface List {
+  goods: string[];
+}
+
+export const GoodList: React.FC<List> = ({ goods }) => (
+  <ul>
+    {goods.map((good, index) => (
+      <GoodCard good={good} key={`id:${index}`} />
+    ))}
+  </ul>
+);
```

Comment:
Ensure that you are not directly manipulating the DOM. React should manage the DOM for you. If there are any direct DOM manipulations, consider refactoring them to use React's state and props. [CODE KNOWLEDGE]